easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.6.3-foss-2016a.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'EB_libQGLViewer'
```

Comment:
no need to hardcode this, if no generic easyblock is specified, EasyBuild will automatically look for an easyblock that matches the software name; so, just remove this line (and the empty one below)